src/docs/product/releases/setup/index.mdx

```diff
@@ -53,14 +53,14 @@ In cases where your code is processed — bundled, minified, transpiled, compile
 - [JavaScript](/platforms/javascript/sourcemaps/)
 - [Node.js](/platforms/node/sourcemaps/)
 
-## 3. Enable Suspect Commits
+## 3. Associate Commits
 
-[Suspect Commits](/product/releases/suspect-commits/) allow you to more easily pinpoint what changes may be responsible for a given issue. To set them up:
+[Associating commits](/product/releases/associate-commits/) to a release allows you to more easily pinpoint which changes may be responsible for a given issue. To set them up:
 
 1. [Install a repository integration](/product/integrations/source-code-mgmt/) so Sentry can access your commit history. (You can also upload commit data manually through our API if you don't want to use an integration.)
 2. Configure your release management script to [associate commits with each release](/product/releases/suspect-commits#enable-suspect-commits). If you're using one of our [release automation](/product/releases/setup/release-automation/) options, this will be done for you.
```

Comment:
```suggestion
2. Configure your release management script to [associate commits with each release](/product/releases/associate-commits). If you're using one of our [release automation](/product/releases/setup/release-automation/) options, this will be done for you.
```